scripts/build-create.sh

```
@@ -9,16 +9,20 @@ VERSION="$1"
 
 conan lock create conanfile.py --version="${VERSION}" --update
 
-conan lock create conanfile.py --version="${VERSION}" --lockfile=conan.lock --lockfile-out=build/conan.lock
-conan create conanfile.py --version "${VERSION}" --lockfile=build/conan.lock --build=missing -o tests=False
+conan lock create conanfile.py --version="${VERSION}" --lockfile=conan.lock --lockfile-out="${WORKDIR}/conan.lock"
+conan create conanfile.py --version "${VERSION}" --lockfile="${WORKDIR}/conan.lock" --build=missing -o tests=False
 
 # Run tests after create
 echo "Running tests..."
-conan test test_package kth/${VERSION} 
+conan test test_package kth/${VERSION}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)